mesonbuild/msetup.py

```diff
@@ -169,16 +172,18 @@ def validate_dirs(self, dir1: str, dir2: str, reconfigure: bool, wipe: bool) ->
         has_partial_build = os.path.isdir(priv_dir)
         if has_valid_build:
             if not reconfigure and not wipe:
-                print('Directory already configured.\n'
-                      '\nJust run your build command (e.g. ninja) and Meson will regenerate as necessary.\n'
+                print('Directory already configured.\n\n'
```

Comment:
`mlog.warning`?

Reply:
I think it deserves to be a warning, and probably to be hard error in the future, because it ignores all -Dfoo options. OTOH, now I'm wondering why having --reconfigure flag at all, that could just be the default behaviour.

Reply:
Added a commit to make --reconfigure the default, that flag is thus useless now.